src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+
+{{ figure_markup(
+  image="performance-LCP-by-rank.png",
+  caption="LCP performance by rank",
+  description="Stacked bar chart showing LCP performance for each ranking group. For the top 1,000 websites, 64% had good, 25% needs improvement, and 11% poor performance. For the top 10,000, 59% were good, 30% needs improvement, and 11% poor. For the top 100,000, 55% were good, 32% needs improvement, and 13% poor. For the top 1,000,000, 53% were good, 33% needs improvement, and 14% poor. For all, 50% were good, 33% needs improvement, and 17% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1080361066&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_rank.sql"
+  )
+}}
+
+For LCP, the differences in performance by rank were closer than FCP. Also, a higher proportion of origins in the top 1,000 had poor LCP. On WordPress, <a hreflang="en" href="https://datastudio.google.com/s/kvq1oJ60jaQ">28% of origins experienced good LCP</a>. This is an opportunity to improve user experience as poor LCP is usually caused by a handful of problems.
+
+### The LCP Element
+
+Let's take a deeper dive into the LCP element.
+
+{{ figure_markup(
+  image="performance-top-15-lcp-nodes.png",
+  caption="Top 15 LCP HTML element nodes",
+  description="Bar chart showing the percent share of pages for each node type for the LCP element- data listed by desktop then mobile: IMG 47% 42%, DIV 28% 27%, P 6% 10%, H1 3% 5%, SECTION 3% 2%, undetected 2% 2%, H2 2% 2%, A 1% 2%, SPAN 1% 1%, HEADER 1% 1%, LI 1% 1%, RS-SBG 1% 1%, H3 0% 1%, TD 1% 1%, VIDEO 0% 0%",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1549429021&format=interactive",
+  width=600,
+  height=909,
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+IMG, DIV, P, and H1 made up 83% of all LCP nodes (on mobile). This doesn't tell us if the content was an image or text, as background images can be applied with CSS.
+
+{{ figure_markup(
+  image="performance-lcp-with-image.png",
+  caption="LCP elements with images, by device",
+  description="Bar chart showing 79% of desktop pages have an LCP element with an image and 71% of mobile pages",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=992354142&format=interactive",
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+We can see that 71-79% of pages had an LCP element that was an image, regardless of HTML node. Furthermore, desktop devices had a higher rate of LCPs as images. This could be due to less real estate on smaller screens pushing images out of the viewport resulting in heading text being the largest element.
```

Comment:
`COUNTIF(elementUrl != "") AS haveImages,`

We don't explain the other queries so that's my only hesitation in explaining it here, but I could go either way.